docs/tutorials/reading-list.md

```diff
@@ -140,6 +140,20 @@ Some titles have been shortened for organizational purposes.
 
 [TypeScript vs. CoffeeScript vs. ES6]: http://www.slideshare.net/NeilGreen1/type-script-vs-coffeescript-vs-es6
 
+## TypeScript
+
+*Tutorial* - [TypeScript in 5 minutes][]
```

Comment:
I think the handbook section (starting from https://www.typescriptlang.org/docs/handbook/basic-types.html) is a lot more useful material than the 5 minutes one, since it mostly focuses on the setup part.

Reply:
Agreed, this is better.